docs/can-guides/topics/debugging.md

```diff
@@ -331,3 +331,99 @@ Component.extend({
+
+## Using the CanJS Devtools ViewModel Editor
+
+When an element is selected in the Elements panel, a CanJS ViewModel Editor will be added in the sidebar:
+
+<img src="../../docs/can-guides/images/devtools/viewmodel-editor.png"
+  alt="The CanJS Devtools ViewModel Editor"
+  width="600px"/>
+
+In order to find this, select the Elements panel:
+
+<img src="../../docs/can-guides/images/devtools/devtools-elements-panel.png"
+  alt="The Chrome Developer Tools Elements Panel"
+  width="600px"/>
+
+Then, depending on the size of your Developer Tools window, you may need to click on the overflow menu in order to find the CanJS ViewModel Editor:
+
+<img src="../../docs/can-guides/images/devtools/devtools-sidebar-overflow-menu.png"
+  alt="The Chrome Developer Tools sidebar overflow menu"
+  width="600px"/>
+
+Once selected, the JSON editor can be used to view and edit the ViewModel for the selected element &mdash; or the closest parent element that has a ViewModel:
+
+<img src="../../docs/can-guides/images/devtools/viewmodel-editor-zoomed.png"
+  alt="The Chrome Developer Tools sidebar overflow menu"
+  width="600px"/>
+
+## Using the CanJS Devtools Bindings Graph
+
+Similar to the ViewModel Editor, when an element is selected, a sidebar is added for the CanJS Bindings Graph:
+
+<img src="../../docs/can-guides/images/devtools/bindings-graph.png"
+  alt="The CanJS Devtools Bindings Graph"
+  width="600px"/>
+
+If the element that is selected has a ViewModel, the graph will show the relationships for a property of the ViewModel:
+
+<img src="../../docs/can-guides/images/devtools/bindings-graph-viewmodel.png"
+  alt="The CanJS Devtools Bindings Graph"
+  width="600px"/>
+
+By default, the property selected will be the _first_ property on the ViewModel alphabetically. You can select a different property by clicking on the header:
+
+<img src="../../docs/can-guides/images/devtools/bindings-graph-viewmodel-click.png"
+  alt="The CanJS Devtools Bindings Graph"
+  width="600px"/>
+
+This will give you a dropdown showing all of the available properties:
+
+<img src="../../docs/can-guides/images/devtools/bindings-graph-viewmodel-select-property.png"
+  alt="The CanJS Devtools Bindings Graph"
+  width="600px"/>
+
+If the element selected does not have a ViewModel, the graph will show the relationships for the element itself:
+
+<img src="../../docs/can-guides/images/devtools/bindings-graph-element.png"
+  alt="The CanJS Devtools Bindings Graph"
+  width="600px"/>
+
+## Using the CanJS Devtools Queues Stack
+
+As discussed in [guides/debugging#Debugwhatcausedaobservableeventorupdatetohappen_ Debug what caused a observable event or update to happen], the `logStack` method of `can-queues` can help figure out what tasks in CanJS's queue system caused a function to run. In order to make this easier, CanJS Devtools adds a sidebar to the Sources Panel to display the `logStack` output:
```

Comment:
Where possible, I’d link to methods in our docs, e.g. `the [can-queues.logStack] method of [can-queues]`

Reply:
And for this particular section, it might be good to embed or link to https://www.youtube.com/watch?v=L0hR5ic_FvE&t=1s

Reply:
I personally would, just because I find it nicer to have links everywhere and it makes it easier for people to start reading a particular section without having to go through all the sections. ¯\_(ツ)_/¯